cmd/controlplane/api.go

```diff
@@ -31,12 +32,12 @@ type apiOptions struct {
 	Host string
 	Port string
 
-	Logger *log.Logger
+	Logger logr.Logger
 }
 
 func newAPICommand() *cobra.Command {
 	cmdOpts := &apiOptions{
-		Logger: log.StandardLogger(),
+		Logger: logging.LoggerFromContext(context.Background()),
```

Comment:
I'm not sure if this is a good way to handle creating a new logger here.

Reply:
@hiddeco and I are working on refactoring `cmd/controlplane` a bit. We'll probably revisit this later, but this line is ok for now.

Reply:
Actually... re-evaluating this, if you could please set this to `log.Log` instead, that would be 👍 

The desire is actually to have "startup" logging be unaffected by the log level selected by whoever installed Kargo. We want info-level messages here, even if the log level is otherwise set to exclude those.

Since `logr` doesn't have a default concrete implementation of the `logr.Logger` interface, the best option here might be to extend our own `logging` package with something like `logging.DefaultLogger()` that would return something like a slog implementation hard-coded to info-level.

This would actually be used by _all_ the `cmd/controlplane` cobra commands.

Reply:
Let me know if something like [74f006b](https://github.com/akuity/kargo/pull/1791/commits/74f006bf5d9c0967717666186a6482f605a5c11c) makes sense. Since `slog.HandlerOptions` is nil here, it defaults to `LevelInfo`, but I don't know how verbose we want to be in the comments. [Docs](https://pkg.go.dev/golang.org/x/exp/slog#HandlerOptions)

Reply:
I've modified the rest of `cmd/controlplane` after the rebase: [aa4336be](https://github.com/akuity/kargo/pull/1791/commits/aa4336be30b3c984dbba124304d9f22995e4b6ee)

Reply:
@bakunowski you got it. Exactly what I wanted -- `LevelInfo` in the initial phases of startup, no matter what the user-configured log level actually is.